app/controllers/admin_controller.rb

```diff
@@ -218,13 +218,6 @@ def fix_results_selector
     render partial: "fix_results_selector"
   end
 
-  def edit_person
```

Comment:
Why does this method have to go?

Reply:
I didn't find any uses in the code for it

Reply:
Then it could certainly go in a different PR, but it feels very out of scope for this current PR